jc/parsers/amixer.py

```diff
@@ -0,0 +1,277 @@
+r"""jc - JSON Convert `amixer sget` command output parser
+
+Usage (cli):
+
+    $ amixer sget <control_name> | jc --amixer
+    $ amixer sget Master | jc --amixer
+    $ amixer sget Capture | jc --amixer
+    $ amixer sget Speakers | jc --amixer
+
+Usage (module):
+
+    import jc
+    result = jc.parse('amixer', <amixer sget command output>)
+
+Schema:
+
+    {
+        "control_name":                     string,
+        "capabilities": [
+                                            string
+        ],
+        "playback_channels": [
+            string
+        ],
+        "limits": {
+            "playback_min":                 integer,
+            "playback_max":                 integer
+        },
+        "mono": {
+            "playback_value":               integer,
+            "percentage":                   integer,
+            "dB":                           float,
```

Comment:
Keys should be all lowercase snake-case, so let's change `dB` to `db` or `decibels`.